frontend/src/container/GridGraphLayout/WidgetHeader/contants.ts

```diff
@@ -0,0 +1,13 @@
+export enum MenuItemKeys {
+	View = 'view',
+	Edit = 'edit',
+	Delete = 'delete',
+	Clone = 'clone',
+}
+
+export enum MenuItemLabels {
```

Comment:
mapping between key and label should be there

Reply:
Okay